tests/mesh_test.cc

```diff
@@ -821,6 +822,31 @@ TEST_CASE("Mesh is checked for 2D case", "[mesh][2D]") {
                   false);
         }
 
+        // Test assign rotation matrices to nodes
+        SECTION("Check assign rotation matrices to nodes") {
+          // Map of nodal id and euler angles
+          std::map<mpm::Index, Eigen::Matrix<double, Dim, 1>> euler_angles;
+          // Euler angles
+          euler_angles.emplace(std::make_pair(
```

Comment:
Here you are assigning only 1 Euler angle per node?

Reply:
no, 2 or 3 depending on dimension

Reply:
Isn't this section Dim 2?

Reply:
Yes it is Dim 2, and we have two angles right?